backend/xivdpscalc/pps/rotation.py

```diff
@@ -0,0 +1,17 @@
+""" High level interface representing rotations to be used by simulators """
+
+class Rotation():
```

Comment:
If this is an abstract class that requires subclasses to implement, I think we should do something like the following:

```python
from abc import ABCMeta, abstractmethod

class Rotation(metaclass=ABCMeta):
    @abstractmethod
    def get_action(self, current_time, cooldowns, remaining_effect_durations, resources):
        ...
```

This forces all subclasses to implement this method (unless you subclass and re-annotate with `@abstractmethod`